chromium_src/components/sync_preferences/DEPS

```diff
@@ -1,3 +1,4 @@
 include_rules = [
+  "+brave/components/email_aliases/browser",
```

Comment:
per https://github.com/brave/brave-core/pull/27127/files#r2014337177 this should be restricted to `:prefs` only

Reply:
but why are we adding this to syncable prefs when it's not even enabled on android? I think we should leave that part out for a separate PR to reduce the size of this one